src/build.sh

```diff
@@ -154,6 +161,16 @@ function kernel_build()
   return "$ret"
 }
 
+# This function adds the build clean option
+# integrated to env in order to clean the kernel.
+function build_clean()
+{
+  local flag="$1"
```

Comment:
Search for the use of `flag` in the code; we usually use it for testing and displaying command messages. You probably want this parameter here, but for another reason. Maybe a better name is `output_kbuild_flag` or `clean_flags`.